.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
[OPTIONAL]
Keep your project professional!!!
- Kindly add to your .gitignore file your .vscode and .idea

Reply:
Hi Carlos i m not sure what i m suppose to do here ?

Reply:
In your root directory, exists external files: .vscode and .idea. 
you need to hide these directories or files into your gitignore.

Reply:
Have added the ideas folder to gitignore as per request